app/Livewire/Admin/LicenseManager.php

```
@@ -0,0 +1,126 @@
+    }
+
+    public function confirmDelete($id)
+    {
+        $this->deleteId = $id;
+        $this->showDeleteModal = true;
+    }
+
+    /**
+     * Delete the order
+     */
+    public function deleteOrderItem($id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (Naming Rules)

(ShortVariable)